### DISCUSSION TITLE
Proper use of Design.DesignStyle

### DISCUSSION THREAD

**Discussion by BytePetter** [AvaloniaUI/Avalonia#15275] [Q&A]
The axaml-Tag

`<Design.DesignStyle>`

does not have proper documentation anywhere on the Avalonia website. It's members suggest it exists to apply a certain style for the designer, but using it just causes the designer to get stuck, and not update with any changes anymore until the tag is removed.

So, how do you properly use it with a CustomControl? An example would be nice.

**Comment by timunie** [comment]
`Design.PreviewWith`. You can look into any style file under src/FluentTheme or SimpleTheme

**Reply by BytePetter** [reply]
1. There is no such thing as src/FluentTheme in the repository. It's actually src/Avalonia.Themes.Fluent.
2. What does Design.PreviewWith has to do with that? I asked about Design.DesignStyle. Is it used? Can it be used by me? If it's not intended to be used by the developer, just say so. 
3. There are no examples of Design.DesignStyle in the style files for any of the controls.

**Reply by timunie** [reply]
@BytePetter I answered from mobile and I thought `PreviewWith` was what you are after, because that is what 99% of the ppl. ask for. Until today I never noticed even that feature. 

Playing around with it I notice you can add this into Window.axaml or anywhere else. Probably useful to test external or dynamic styles in the previewer

```xml
<Window ...>
    <Design.DesignStyle>
        <Style Selector="TextBlock">
            <Setter Property="FontSize" Value="100" />
        </Style>
    </Design.DesignStyle>
    <Content />
</Window>
```
